meta-avocado/recipes-avocado/distro/avocado-complete.bb

```diff
@@ -2,13 +2,9 @@ SUMMARY = "Meta-target to build Avocado OS images, extensions, and extra package
 LICENSE = "Apache-2.0"
 
 # Ensure compile task runs after dependencies
-do_compile[depends] += "avocado-image-initramfs:do_image_complete"
-do_compile[depends] += "avocado-image-rootfs:do_image_complete"
-do_compile[depends] += "avocado-image-var:do_deploy"
-do_compile[depends] += "${@bb.utils.contains('MACHINE_FEATURES', 'genimage', 'avocado-image-genimage:do_deploy', '', d)}"
```

Comment:
You just added this for `genimage` in your last PR. Why are you taking it away?

Ah, you moved it to `avocado-core.bb`.